app/plugin_api.go

```diff
@@ -160,6 +160,10 @@ func (api *PluginAPI) GetUserByUsername(name string) (*model.User, *model.AppErr
 	return api.app.GetUserByUsername(name)
 }
 
+func (api *PluginAPI) GetUsersByUsernames(usernames []string, asAdmin bool) ([]*model.User, *model.AppError) {
```

Comment:
I think it makes sense to remove the parameter for `asAdmin` and just always use true in the app method call. All it's doing is sanitizing the returned profiles which plugins can do themselves if needed and I'd rather keep the API simpler where possible.

@lieut-data @czertbytes thoughts?

Reply:
I am very new to pluginAPI but generally it makes sense to me to remove `asAdmin` parameter and use to `true` in method call. 👍

Reply:
100% agree that we should simplify the API here, and there's no compelling reason to get users as a non-admin at present.